README.md

```diff
@@ -103,27 +142,34 @@ You should end up with a structure like the following for the `test_discovery.py
 └───greenkey-discovery-sdk
     └───discovery_binaries_windows_10_64bit__python37_64bit
     └───examples
-    └───gkcli
+    └───gk_cli
     │   discovery_config.py
     │   README.md
     │   test_discovery.py
 ```
 
-### Using the Discovery CLI tool
 
-The Discovery CLI is a tool that can aid in the creation of custom definition files for use with Discovery.
+# 2. Customization and Discovery CLI
+Creating a custom project can be done by following the structure of an existing example found in `examples/`. The Discovery CLI tool can also be used to guide you through creating your own project and custom definition files for use with Discovery.
```

Comment:
nitpicky... change to `examples folder` so it matches how you reference directories everywhere else